### PR TITLE
Setup continuous integration and coverage report

### PR DESCRIPTION
**Pull request by jyuw**
#### Description
Changes proposed in this pull request:

* Added the travis yaml file
* Added Coveralls gem

#### What I have learned working on this feature: 
How to add Travis and Coveralls

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: ruby
+cache: bundler
+rvm:
+  - 2.4.1
+script:
+  - bundle install
```
Comment (diraulo) [inline .travis.yml]: You don't need to do `bundle install` Travis will do that automatically for every ruby projects

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - bundle exec rake db:create --all
```
Comment (diraulo) [inline .travis.yml]: Move all DB Setup in a `before_script:` block.
Also you don't need the `--all` of creating the DB and you can compact this in one line

`bundle exec rails db:create db:test:prepare` should do the trick in principle

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - bundle exec rake db:migrate
+  - bundle exec rake db:test:prepare
+  - bundle exec rake
+services:
+  - postgresql
+addons:
+  chrome: stable
+install:
```
Comment (diraulo) [inline .travis.yml]: Doesn't make any difference to Travis but IMO it makes sense to order these in the order in which they're actually executed.

So I'd move this step above before the `script` and `before_script`

**Review by diraulo — CHANGES_REQUESTED** [review]
👎

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  - wget -N http://chromedriver.storage.googleapis.com/2.33/chromedriver_linux64.zip -P ~/
+  - unzip ~/chromedriver_linux64.zip -d ~/
+  - rm ~/chromedriver_linux64.zip
+  - sudo mv -f ~/chromedriver /usr/local/share/
+  - sudo chmod +x /usr/local/share/chromedriver
+  - sudo ln -s /usr/local/share/chromedriver /usr/local/bin/chromedriver
+before_script:
+  - bundle exec rails db:create db:test:prepare
+  - bundle exec rake
```
Comment (diraulo) [inline .travis.yml]: This line actually executes the test and should be in the `script`  block

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+script:
+services:
```
Comment (diraulo) [inline .travis.yml]: Move this before the `install` block

**Review by diraulo — CHANGES_REQUESTED** [review]
Minor changes and I think we can get this in if it actually passes on Travis. I just enabled Travis for this repo so the next commit you push should trigger a build.

**Comment by diraulo** [comment]
You should merge the `master` branch onto this one to have it in sync - I see the logs are back on branch. Once that's done we can merge this in.

**Review by diraulo — CHANGES_REQUESTED** [review]
Please remove the log files

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  - 2.4.1
+addons:
+  chrome: stable
+before_script:
+  - bundle install
+  - bundle exec rails db:create db:test:prepare
+script:
+  - bundle exec rake
+install:
+  - wget -N http://chromedriver.storage.googleapis.com/2.33/chromedriver_linux64.zip -P ~/
```
Comment (tochman) [inline .travis.yml]: I don’t understand. Why are you installing Chrome on Travis? You are not using Chromedriver in your tests.

**Review by diraulo — CHANGES_REQUESTED** [review]
I was about to approve this PR as agreed in Scrum.. but now I'm gonna have to reject it. This Feature is only about setting up Travis. You should deal with coveralls on a separate PR

**Comment by diraulo** [comment]
@jyuw Okay, I see you've added Coveralls to the story on PT. You should probably change the title of this PR to reflect what you're doing. eg `Setup continuous integration and coverage report`

As for Coveralls you're only setting it up for RSpec. What about Cucumber? And also you should probably merge both reports

**Comment by jyuw** [comment]
The Pivotal Tracker story has both Travis and Coveralls in it, is that not allowed?

**Comment by diraulo** [comment]
You forgot to exclude the `coverage` folder from version control 🤔

**Review by diraulo — APPROVED** [review]
👍

**Pull request merged**